### DISCUSSION TITLE
Deta Base SDK - fs error

### DISCUSSION THREAD

**Discussion by malwaremanu** [deta/cloud-docs#633] [Help]
I Don't know where exactly i am doing wrong.

ERROR in ./node_modules/deta/dist/index.browser.js
Module not found: Error: Can't resolve 'fs' in '~\node_modules\deta\dist'

Here's my package.json

`{
  "name": "inotes",
  "version": "1.0.0",
  "private": true,
  "scripts": {
    "dev": "nuxt",
    "build": "nuxt build",
    "start": "nuxt start",
    "generate": "nuxt generate",
    "lint:prettier": "prettier --check .",
    "lint": "npm run lint:prettier",
    "lintfix": "prettier --write --list-different ."
  },
  "dependencies": {
    "core-js": "^3.19.3",
    "deta": "^1.1.0",
    "nuxt": "^2.15.8",
    "vue": "^2.6.14",
    "vue-server-renderer": "^2.6.14",
    "vue-template-compiler": "^2.6.14",
    "webpack": "^4.46.0"
  },
  "devDependencies": {
    "@nuxtjs/tailwindcss": "^4.2.1",
    "eslint-config-prettier": "^8.3.0",
    "postcss": "^8.4.4",
    "prettier": "^2.5.1"
  }
}
`

Please help.

**Comment by ejnalo** [comment]
Nobody here ? I have the same problem.
